tests/extract_run_test.sh

```diff
@@ -27,5 +28,5 @@ mkdir "$lab"
 cd "$lab" || exit
 
 # copy sample unit tests and execute
-cp -rf ../../../../docs/examples/unit_tests/* .
+cp -rf ../../../../../docs/examples/unit_tests/* . || exit
```

Comment:
why is this change necessary?

Reply:
Since we have execution_node_1, ... , execution_node_5 inside execution_nodes directory as per the new structure, while the extract_run.sh still remains in the respective execution_node_1, ...  , execution_node_5 directories, this change is required. The additional || exit has been removed.